src/components/CodeMirrorComponent.js

```diff
@@ -274,6 +274,7 @@ export default function CodeMirrorComponent(props) {
           autoCloseBrackets: true,
           lineNumbers: true,
           foldGutter: true,
+          readOnly: props.usage ? true : false,
```

Comment:
This is pretty minor, but I'm wondering if either the name of this prop should be updated or the check should be updated to make the code a little clearer. I think the idea here is that if the editor is used for examples, it should be a read-only editor. Since the prop is technically saying that there is a usage and that usage is 'examples', do you think it makes sense to have this check be something like this?

```
readOnly: props.usage === 'examples' ? true : false
```

The other possibility that I thought of is swapping the usage prop to just be a boolean like `props.isExamples`, and then the check here could stay as is. I think either change might give a little more context to this line within the component itself.

Reply:
This is a much better approach, I implemented this in commit `5e25d7a`.

Reply:
Awesome! I like it!